### ISSUE TITLE
Add build instructions for quarto-web to README

### ISSUE THREAD

**Issue by cwickham** [quarto-dev/quarto-cli#4496]
For example:

* What version of quarto to use: dev

* To build site locally:
    ```
    quarto render
    ```

* How to generate reference section (based on assumptions about location of `quarto-cli`:
    ```
    quarto run tools/reference.ts
    ```

* Actions that publish website (I'm not familiar with these yet, but would be useful to document roughly when and how website is published)

**Comment by cwickham** [comment]
Include a description of the process by which previews for PRs are generated quarto-dev/quarto-web/pull/667

**Comment by cwickham** [comment]
Also include the people/orgs that have access to Netlify

**Comment by mcanouil** [comment]
At work, I have CI that check only the files in a PR to perform some actions on them.
Would it be worth it to add a PR command GHA, that perform `quarto render` on qmd files from a PR?

**Comment by cderv** [comment]
I believe you are discussing the last point I shared - IMO this would be better in another thread discussing a change of workflow. Anyway, everything is possible yes with Github action. We could indeed do that - I have thought of that too. Though, this would mean changing the workflow, and maintain some computation environment in the CI (which we don't for quarto web at the moment). For now, we are avoiding setting and maintaining R, python or Julia env there. 

After thinking about it, I found that using `freeze: true` workflow with local rendering was the easiest for the website and it was just a matter of adapting my workflow for quarto-web contribution. 

Happy to adapt the action if we think that is worth it, but sometime less maintenance burden is also good 😄
